plugins/com.google.cloud.tools.eclipse.appengine.facets.test/src/com/google/cloud/tools/eclipse/appengine/facets/ui/AppEngineStandardProjectConvertCommandHandlerTest.java

```diff
@@ -113,6 +115,8 @@ public void testCheckFacetCompatibility_java1_6FacetIsIncompatible() throws Core
 
   @Test
   public void testCheckFacetCompatibility_java1_8FacetIsIncompatible() throws CoreException {
+    Assume.assumeTrue("AppEngine Standard Java 8 runtime support is not present",
```

Comment:
The Java 1.8 facet is not incompatible when the .standard.java8 bundle is available.  Tycho will run with the minimal bundles required, and so the test should pass.  But the test will likely fail when run in Eclipse since _JUnit Plug-in Tests_ defaults to running with all bundles.